src/appleseed-max-impl/appleseedlightmtl/appleseedlightmtl.cpp

```diff
@@ -149,6 +150,12 @@ namespace
             p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_EDIT_LIGHT_POWER, IDC_SPINNER_LIGHT_POWER, SPIN_AUTOSCALE,
         p_end,
 
+        ParamIdLightExposure, L"light_exposure", TYPE_FLOAT, P_ANIMATABLE, IDS_LIGHT_EXPOSURE,
+            p_default, 0.0f,
```

Comment:
Default should be 1.

Reply:
I don't think it's a good idea. This would make all scenes previously setup with `appleseedlightmtl` twice as bright. Also value of 1 seems rather arbitrary, value of 0 is neutral and as such has special meaning.
Having said that, if you feel this is still needed for consistency or otherwise, I'll gladly change that.

Reply:
Doh, of course you're right. Facepalm :)